swagger.yaml

```diff
@@ -30,6 +26,22 @@ paths:
           schema:
             $ref: '#/definitions/Info'
 
+  /departments:
+    get:
+      summary: List departments
+      description: Return org code and org description (departments) for a business center
+      parameters:
+        - in: query
+          name: business_center
```

Comment:
In the query, it's business_center, but in the response, its bcName. I think these two things should be the same, preferably an option without underscores since most of our APIS use camel case query parameters. Maybe businessCenter? (could be case insensitive in the query)